src/test/java/de/tum/in/ase/parser/IntegrationTest.java

```diff
@@ -16,69 +21,97 @@
-
-    private static final String EXPECTED_INVALID_FILENAME = "{\"tool\":null,\"issues\":[{\"filePath\":\"cpd_invalid.txt\",\"startLine\":1,\"rule\":\"ExceptionDuringParsing\",\"category\":\"miscellaneous\",\"message\":\"An exception occurred during parsing the report for file cpd_invalid.txt. Exception: java.lang.IllegalArgumentException: File must be xml format\"}]}";
-
-    private static final String EXPECTED_INVALID_XML = "{\"tool\":null,\"issues\":[{\"filePath\":\"invalid_xml.xml\",\"startLine\":1,\"rule\":\"ExceptionDuringParsing\",\"category\":\"miscellaneous\",\"message\":\"An exception occurred during parsing the report for file invalid_xml.xml. Exception: %s\"}]}";
-
-    private static final String EXPECTED_INVALID_NAME = "{\"tool\":null,\"issues\":[{\"filePath\":\"invalid_name.xml\",\"startLine\":1,\"rule\":\"ExceptionDuringParsing\",\"category\":\"miscellaneous\",\"message\":\"An exception occurred during parsing the report for file invalid_name.xml. Exception: de.tum.in.ase.parser.exception.UnsupportedToolException: Tool for identifying tag data not found\"}]}";
+        ReportParser parser = new ReportParser();
+        String actual = parser.transformToJSONReport(toolReport);
+
+        try {
+            BufferedReader reader = Files.newBufferedReader(Paths.get(EXPECTED_FOLDER_PATH + expectedJSONReportFileName));
+            String expected = reader.readLine();
```

Comment:
```suggestion
            String expected = reader. lines()       .collect(Collectors.joining(System.lineSeparator()));
```
Allows us to trad multi line strings in the future.